km/km_exec.c

```diff
@@ -529,74 +359,19 @@ char** km_exec_build_argv(char* filename, char** argv, char** envp)
 {
```

Comment:
See above: I think we can get rid of km_is_shell_path() and km_is_env_path() and km_one_of() and 
static const_string_t SHELL_PATHS[] =
static const_string_t ENV_PATHS[] =

And the function prototypes in km/km_exec.h

And remove the comment about trying to run the shell in the block comment above.

Hope I'm not missing some reason to keep these around.